frontend-react/src/views/Drives/components/DrivesList/mockItems.js

```diff
@@ -0,0 +1,2804 @@
+export const mockDrives = [
```

Comment:
The data format is not consistent with Drives format returned from the backend. Properties names should be in camelCase and data for nested objects (e.g. projects) should not be flattened. Mocking data in the final format allows you to avoid further changes in components when real data will be used.

Reply:
I think that the best source will be to check in the browser how the data is returned by this endpoint for the vue application